File: Models/Hotel.cs

namespace Models
{
    public class Hotel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string MetaDescription { get; set; }
        public Address AddressHotel { get; set; }
        public string Email { get; set; }
        public int CodeArea { get; set; }
        public int PhoneNumber { get; set; }

        //public static implicit operator Hotel(Infrastructure.Models.Hotel hotel)
        //{
        //    return new Hotel()
        //    {
        //        Id = hotel.Id,
        //        Name = hotel.Name,
        //        Description = hotel.Description,
        //        MetaDescription = hotel.MetaDescription,
        //        AddressHotel = hotel.AddressHotel,
        //        Email = hotel.Email,
        //        PhoneNumber = hotel.PhoneNumber,
        //        CodeArea = hotel.CodeArea,
        //    };
        //}

    }
}

File: ClientApp/ClientApp/src/models/hotel.ts
import { Address } from "./Address";
import { hotelPicture } from "./hotelPicture";

export interface Hotel {
  id: number;
  name: string;
  description: string;
  metaDescription: string;
  addressHotel: Address
  email: string;
  codeArea: number;
  phoneNumber: number;
  postalCode: string;
  latitud: string;
  longitud: string;
  image: string;
  pictures: hotelPicture[]

}
